### DISCUSSION TITLE
Pin a post on home page!

### DISCUSSION THREAD

**Discussion by blogi-kriptovalyut** [mmistakes/minimal-mistakes#2675] [Ideas]
Hey. I got an idea.
## Summary
It would be great to do not only a recent post on home page, but also a "Pinned Posts".
## Motivation
Pinning a post is one of the easiest - and most effective - ways to ensure that your best and most important post is the first thing visitors see when they land on your home page.
You can use something like this:
`{% assign pinned = site.posts | where_exp: "item", "item.pin == true"%}`

We can create a yml variable in each post called `pin`, and set some to true.

So I see a very nice and informative home page like this:

![img-2020-08-29-13-48-11](https://user-images.githubusercontent.com/50487083/91635117-5d082000-e9fe-11ea-84f2-d93994bea316.png)

**Comment by mmistakes** [comment]
Duplicate of #2045
